api/apiroutes.go

```diff
@@ -1560,11 +1560,12 @@ func (c *appContext) getAddressTxAmountFlowData(w http.ResponseWriter, r *http.R
 
 func (c *appContext) ChartTypeData(w http.ResponseWriter, r *http.Request) {
 	chartType := m.GetChartTypeCtx(r)
-	zoom := r.URL.Query().Get("zoom")
-	chartData, err := c.charts.Chart(chartType, zoom)
+	bin := r.URL.Query().Get("bin")
```

Comment:
I renamed most of all usages of the word and variable `zoom` to `bin` to reduce confusion from the zoom front-end charts feature.

Reply:
I'm disinclined to make such changes since it breaks existing links.  How about supporting "zoom" of no "bin" is provided, and then redirecting to the link with "zoom"?

Reply:
I've used those words a little too liberally, for sure.  I think my thinking was that `zoom`  would correspond to a `zoom level`, and some day in the future, the default bin sizes returned might depend on how far the user is zoomed in, à la google maps. But it's not that yet, so it's probably just a little confusing for now. 

@chappjc's suggestion sounds good for now, but can we mark the date in the docs and give a deprecation warning in the js console so that we can eventually simplify things?

Reply:
I have added support for the deprecated "zoom" URL parameter. I think the change might be to small to warrant a deprecation warning on the front-end. Also the previous usage of "zoom" URL parameter is easily supported without making too many adjustments.